ui.js

```diff
@@ -0,0 +1,40 @@
+
+const PLAY_ICON = "play_arrow";
+const PAUSE_ICON = "pause";
+
+class UI {
+  constructor(config) {
+    this.game = config.game;
+    this.playButton = config.playButton;
+    this.stepButton = config.stepButton;
+    this.stopButton = config.stopButton;
+    this.generationCounter = config.generationCounter;
+    this.events = config.events;
+    //this.mousePosition = config.mousePosition;
+    this.initUI();
+  }
+
+  initUI() {
```

Comment:
you could be more specific and call these set of actions as registering event handlers